### PR TITLE
0.10 blog post.

### PR DESCRIPTION
**Pull request by grokys**
**What kind of change does this PR introduce?**

Blog post for 0.10 release and related docs.

### CODE REVIEW

File: src/AvaloniaUI.Net/wwwroot/blog/2020-12-29-avalonia-0.10.0-release.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+Related PRs:
+
+- https://github.com/AvaloniaUI/Avalonia/pull/4904
+
+## Compiled Bindings
+
+Avalonia 0.10 includes experimental support for compiled bindings. When using compiled bindings, binding paths are verified at compile time and do not use reflection at runtime.
+
+To enable compiled bindings, add an `x:DataType` attribute to your root control and use the `{CompiledBinding}` markup extension or set `x:CompileBindings="True"`.
```
Comment (jkoritzinsky) [inline src/AvaloniaUI.Net/wwwroot/blog/2020-12-29-avalonia-0.10.0-release.md:77]: We should add something hear about using ReflectionBinding when x:CompileBindings=true to use an old style binding.

Comment (grokys) [reply src/AvaloniaUI.Net/wwwroot/blog/2020-12-29-avalonia-0.10.0-release.md:77]: Thanks, done!

File: src/AvaloniaUI.Net/wwwroot/docs/quickstart/logging.md [hunk 2]
```diff
@@ -32,23 +30,11 @@ by the members of `Avalonia.Logging.LogArea` static class and are currently:
 - `Layout`
 - `Control`
 
-The `LogToDebug` method doesn't currently allow specifying an area to display,
-so you need to set up the serilog pipeline manually and use a filter. This can
-be done using the following code:
+The `LogToTrace` method allows specifying which areas are logged:
 
 ```csharp
-SerilogLogger.Initialize(new LoggerConfiguration()
```
Comment (maxkatz6) [inline src/AvaloniaUI.Net/wwwroot/docs/quickstart/logging.md:40]: I believe we should add some short information about implementing custom Sink logger.

Comment (grokys) [reply src/AvaloniaUI.Net/wwwroot/docs/quickstart/logging.md:40]: Yeah, probably but that's an advanced topic and not really related to 0.10 so will punt on it for now.

**Pull request merged**